lib/ansible/module_utils/network/cnos/cnos.py

```diff
@@ -2187,29 +2209,31 @@ def bgpConfig(
 # EOM
 
 
-def vlanConfig(
-    obj, deviceType, prompt, timeout, vlanArg1, vlanArg2, vlanArg3,
-        vlanArg4, vlanArg5):
+def vlanConfig(module, prompt, answer):
 
     retVal = ""
     # Wait time to get response from server
-    timeout = timeout
+    timeout = 2
```

Comment:
I don't see timeout here anymore, is it still used? Why 2?

Reply:
when using paramiko, you can always specify a timeout to regard the switch as not responding. This code is a left over of legacy code. Not relevant now, will remove.

Reply:
Removed